Emby.Server.Implementations/ScheduledTasks/Tasks/AudioNormalizationTask.cs

```diff
@@ -195,7 +195,8 @@ await foreach (var line in reader.ReadAllLinesAsync(cancellationToken))
 
                 if (match.Success)
                 {
-                    return float.Parse(match.Groups[1].ValueSpan, CultureInfo.InvariantCulture.NumberFormat);
+                        await process.WaitForExitAsync(cancellationToken);
```

Comment:
```suggestion
                        await process.WaitForExitAsync(cancellationToken).ConfigureAwait(false);
```

Reply:
I tried to make that change, but now the debug logs(calculating....) are not shown in console. Can you check?